### ISSUE TITLE
"Make this function static" missing space

### ISSUE THREAD

**Issue by asgraf** [hammer-tools/php-hammer#29]
Example code 1:
```php
$this->someFunction('Argument1', function () {
    return true;
});
```
Applying "Make this function static" quick fix results in: 
```php
$this->someFunction('Argument1', staticfunction () {
    return true;
});
```

Example code 2:
```php
$this->someFunction('Argument1', fn() => true);
```
Applying "Make this function static" quick fix results in: 
```php
$this->someFunction('Argument1', staticfn() => true);
```

In both examples there is missing space after `static` keyword

**Comment by asgraf** [comment]
Sorry for the late reply, I was on vacation.
Please try to put my example code inside class method

**Comment by rentalhost** [comment]
It still works for me, but I've seen it happen in some cases (in Blade files, for example). Could you share your PhpStorm version and perhaps the formatting setting? So I can try to use your settings and try to simulate the problem again.

![image](https://github.com/hammer-tools/php-hammer/assets/421851/57162951-3ca5-4415-aced-578a9180b13e)
![image](https://github.com/hammer-tools/php-hammer/assets/421851/3fe673bb-fea6-4650-bc45-ab05cd623e9f)

**Comment by fMads** [comment]
I have the same issue. Here I've tested with your code example

![hammer static function before](https://github.com/hammer-tools/php-hammer/assets/832842/774232e3-37d4-47ea-801b-bf1d7148c842)
![hammer static function after](https://github.com/hammer-tools/php-hammer/assets/832842/4188ecb9-ca86-4118-9e96-a814e39dc135)

And my PhpStorm configuration
```
PhpStorm 2023.2.1
Build #PS-232.9559.64, built on August 25, 2023
Licensed to Spy Aps / Mads Føhns
Subscription is active until January 1, 2024.
Runtime version: 17.0.8+7-b1000.8 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 8192M
Cores: 16
Registry:
    ide.browser.jcef.enabled=false
    debugger.watches.in.variables=false
    debugger.new.tool.window.layout=true
    run.processes.with.pty=TRUE
    ide.experimental.ui=true

Non-Bundled Plugins:
    Batch Scripts Support (1.0.13)
    com.jinsihou.react.snippets (1.1.1)
    com.github.copilot (1.3.2.3479)
    org.semonte.intellij.swagger (1.0.40)
    dev.meanmail.plugin.nginx-intellij-plugin (2022.1.1)
    com.intellij.kubernetes (232.9559.34)
    com.intellij.grazie.pro (0.3.256)
    net.rentalhost.plugins.php.hammer (0.24.0)
    com.haulmont.rcb (2023.3-232)
``` 

I'm not sure if the Code Style has anything to do with this - but here it is
[Code style.zip](https://github.com/hammer-tools/php-hammer/files/12913854/Code.style.zip)

**Comment by rentalhost** [comment]
@fMads nice! I could reproduce the issue using your Code Style. Now I will debug to understand if it is a bug on plugin or PS at all.

**Comment by rentalhost** [comment]
I [**reported it**](https://jetbrains-platform.slack.com/archives/C5U8BM1MK/p1697481911139369) to the Jetbrains Platform Slack channel because, even though I was able to reproduce it directly in the IDE, as mentioned earlier, I couldn't reproduce it through unit tests. This likely indicates that it's a problem with their SDK. I'll wait for their response and update here as soon as I have more information.

**Comment by rentalhost** [comment]
Thank you for your time in reporting this issue. The fix has been implemented in **version 0.25.0** and is **available exclusively** through the [**Releases**](https://github.com/hammer-tools/php-hammer/releases) page. It will be implemented in the JetBrains Marketplace once [**this issue**](https://github.com/hammer-tools/php-hammer/issues/10) that are affecting some users are resolved (no ETA).

**In response to your report:**

- The issue has been identified and possibly fixed. Now, when applying the suggested fix, a space will be added, regardless of the currently used code style configuration.
